recipes/phasefieldx/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "phasefieldx" %}
+{% set version = "0.0.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/CastillonMiguel/phasefieldx/archive/v{{ version }}.tar.gz
+  sha256: f85113793ba1f199647f715a94ee5f36cc96f6288aef0dbe227812e8e7b70402
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+  skip:
+    - win-64
```

Comment:
```suggestion
  skip: True  # [win]
```

Reply:
I don’t know why, but I got an error when I changed to the suggestion, so I reverted to the last version.

Reply:
What error? This seems to be skipping everything.

Reply:
I have added the suggested changes again, but I am getting this error (if I remove noarch, I also get errors)

For recipes/phasefieldx/meta.yaml:

noarch packages can't have skips with selectors. If the selectors are necessary, please remove noarch: python.

Reply:
> I have added the suggested changes again, but I am getting this error (if I remove noarch, I also get errors)

For clarity the errors in the logs are:

```
Encountered problems while solving:
  - nothing provides requested fenics-dolfinx 0.8.0.*
  - nothing provides requested mpich
  - nothing provides requested petsc4py
  - nothing provides requested time
```

It is OK for noarch package to fails on Windows with a missing dependency. We prefer noarch b/c it is faster to build and the package will be ready when the missing dependency is available on Windows.